big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * 1 = O(1)
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 2 * n = O(2n)
+    i++                             // 2 * n = O(2n)
+  ) {
+    const word = words[i];          // 1 * 1 = O(1)
```

Comment:
This is 2 simple actions, and is still within the `for` loop so it would be `O(2n) ~= O(n)`